src/ampState.js

```diff
@@ -0,0 +1,10 @@
+export default function ampState(data) {
+  if (!data) return '';
+  return Object.entries(data).map(([key, value]) => (
+    `<amp-state id="${key}">
```

Comment:
This is valid, it uses custom elements. More info: https://developers.google.com/web/fundamentals/getting-started/primers/customelements